cmd/fyne/internal/commands/build.go

```diff
@@ -277,7 +291,10 @@ func (b *Builder) build() error {
 		return err
 	}
 
-	b.runner.setDir(b.srcdir)
```

Comment:
I agree that abspath is a good update - but can it still be passed to `setDir` so we don't change the semantics of path lookups inside the runner?

Reply:
done